.travis.yml

```diff
@@ -15,15 +15,18 @@ before_install:
       wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
     fi
   - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
+  - source $HOME/miniconda/etc/profile.d/conda.sh
+  - conda activate base
```

Comment:
This is by far the preferred way of activating a conda environment, not setting the `PATH`.  Certain environment variables don't get set if `conda activate` is not called.